src/Http/Session/DatabaseSession.php

```diff
@@ -122,14 +122,20 @@ public function read($id)
             ->first();
 
         if (empty($result)) {
-            return false;
+            return '';
         }
 
         if (is_string($result['data'])) {
             return $result['data'];
         }
 
-        return stream_get_contents($result['data']);
+        $session = stream_get_contents($result['data']);
+
+        if ($session === false) {
+            return '';
```

Comment:
This case might need to return false since it's an actual read failure.

Reply:
Lets keep the `''`. Failing to start the session won't help the end user. This scenario looks like we found the session record but its data is corrupted, in that scenario we should probably start a new session as blowing up the request won't help the end user move forward at all.